front/packages/ui/src/home/recommanded.tsx

```diff
@@ -89,7 +90,7 @@ export const ItemDetails = ({
 					height: ItemDetails.layout.size,
 					flexDirection: "row",
 					bg: (theme) => theme.variant.background,
-					borderRadius: 6,
+					borderRadius: 12, // to make the border stick to the poster
```

Comment:
I don't understand why you are not using the constant for image radius. Is it because you need to add borderWidth to borderRadius ? (if that's the case, why css, why).

If that's the case, you can use `calc(px(ImageRadiusConst), "+", ts(.25))` using yoshiki/native

Reply:
Or you could just set it to `borderRadius: ImageRadiusConst` and set the poster's style to `borderRadius: 0` instead, you already set borderRight to 0 anyways.